src/routes/news/components/NewsElement.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import { LocalNewsModel } from '@integreat-app/integreat-api-client'
+import CleanLink from '../../../modules/common/components/CleanLink'
+import LastUpdateInfo from './../../../modules/common/components/LastUpdateInfo'
+import type { TFunction } from 'react-i18next'
+import { textTruncator } from './../../../modules/theme/constants/helpers'
+
+const TRUNCATE_LETTERS_COUNT = 30
+
+const Link = styled(CleanLink)`
+  display: flex;
+  background-color: white;
```

Comment:
Please don't use colors like this but instead use those defined in the theme.

Reply:
What if the color is not defined in the theme and used once or twice?

Reply:
Fixed as you are using background color now, right?

Reply:
Yeah